dace/codegen/CMakeLists.txt

```diff
@@ -85,6 +87,9 @@ foreach(DACE_FILE ${DACE_FILES})
       set(DACE_RTL_FILES ${DACE_RTL_FILES} ${DACE_FILE})
     elseif(DACE_FILE_EXT MATCHES ".cpp")
       set(DACE_HOST_FILES ${DACE_HOST_FILES} ${DACE_FILE})
+    # TODO I'm not sure the tcl files are needed in CMake?
```

Comment:
Not currently, as they are put in the same folder as the RTL code. E.g. for `rtl_tasklet_1.sv` there will be a `rtl_tasklet_1_package.tcl`. Since they have to be linked to each RTL kernel, I think this is the best way to handle it. Otherwise it has to be matched up to the RTL sources anyways. I just kept it there to remind me to double check once the work is done.

Reply:
Did you double check this? Do we still need it?

Reply:
Yes, and no. They've been removed in https://github.com/carljohnsen/dace/commit/9224152e6c75a4a071a6e36abc296515ddc44cf3